<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1774</width>
    <height>1223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>194</width>
      <height>22</height>
     </rect>
    </property>
    <property name="dateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>11</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </datetime>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </datetime>
    </property>
    <property name="maximumDate">
     <date>
      <year>2014</year>
      <month>12</month>
      <day>31</day>
     </date>
    </property>
    <property name="minimumDate">
     <date>
      <year>2013</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>doors</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_door">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="doorTable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="fanTable">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="itemTable">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>90</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="motionTable">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>90</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>fan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>item</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>motion</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="lightTable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>610</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tempTable">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>610</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="battTable">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>610</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="systemTable">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>610</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_fan">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_item">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_motion">
    <property name="geometry">
     <rect>
      <x>1380</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_5">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>590</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lights</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_light">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_temp">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>query datetime</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>590</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>590</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>battery percetage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>590</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>system</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_batt">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_refresh_system">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1774</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
